python-main.js

```diff
@@ -821,6 +821,21 @@ function web_editor(config) {
         }
     }
 
+    function showWarning(check){
```

Comment:
If this is only called inside `invalidExt`, let's move it inside that function.

Reply:
I'm going to call this when I'm checking for makecode hex files so I'll leave it as is for now

Reply:
In that case can we rename the function? "showWarning" is very generic (I would expect a function with this type of name to show any type of warning, perhaps similarly to what `modalMsg()` does now). The code in this function is very specific to warnings related to a loaded file extension, so `invalidExt()` is a better name for this one?

Reply:
does `invalidFileWarning()` work? If its going to be used to show warnings for invalid file extensions and invalid hex files

Reply:
Yeah, sounds good.